packages/desktop-client/src/components/SidebarWithData.js

```diff
@@ -51,9 +51,10 @@ function EditableBudgetName({ prefs, savePrefs }) {
   }
 
   let items = [
-    { name: 'rename', text: 'Rename Budget' },
+    { name: 'rename', text: 'Rename budget' },
+    { name: 'close', text: 'Close file' },
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏it _feels_ more natural to have the "close" option as the very last one. But maybe it's just me.. 🤷